base

```diff
@@ -0,0 +1 @@
+.
```

Comment:
It was intended but I've made sure just now that we can do without this symbolic link.